frame/staking/src/tests.rs

```diff
@@ -4013,4 +4023,130 @@ mod election_data_provider {
 			assert_eq!(ForceEra::<Test>::get(), Forcing::NotForcing);
 		})
 	}
+
+	#[test]
+	#[should_panic]
+	fn count_check_works() {
+		ExtBuilder::default().build_and_execute(|| {
+			// We should never insert into the validators or nominators map directly as this will
+			// not keep track of the count. This test should panic as we verify the count is accurate
+			// after every test using the `post_checks` in `mock`.
```

Comment:
```suggestion
			// after every test using the `post_conditions` in `mock`.
```

didn't find `post_checks`, but I think this is what you meant